### DISCUSSION TITLE
2021-12-15 Weekly Meeting 34 Agenda / Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#147] [Weekly Meetings]
Weekly Meeting 34 is happening on Wednesday December 15th, 2021 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Agenda

   - Welcome new members and attendance
   - Upcoming events, next demo night 
   - We are refining our working groups, more info next week 
   - [Lean coffee](https://trello.com/invite/b/R7l2LPsW/1539a4b65556e61ba2ecb4ce0255c824/lean-coffee) open discussion

**Comment by mrmetaverse** [comment]
New format for today, but we will resume next week.

**Comment by mrmetaverse** [comment]
Discussed our next Demo Night, looking at first week of January 2022. 

4-5 day sweet spot before CES. 

Our last demo night can be viewed [on youtube here](https://www.youtube.com/watch?v=e26UJRCGfGk)

3-5 demos, will open up a form when we decide on a date.

**Reply by RedEagleP1** [reply]
Loved lean coffee.

**Comment by mrmetaverse** [comment]
Other upcoming events? 

- https://www.theculturedao.com/ - meeting today

**Comment by mrmetaverse** [comment]
#130 - do we have an owner for the shared calendar effort?

**Reply by mrmetaverse** [reply]
We need a partner to help see through the execution of a shared event calendar on our website. 

We already have a OMI event calendar here: https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t

**Reply by mrmetaverse** [reply]
This would be a separate shared event calendar.

**Comment by mrmetaverse** [comment]
# WASM group next steps [og ticket](https://github.com/omigroup/omigroup/issues/133)
create github repo (humbletim) checkered_flag
create discord thread
- form working agreement and add to README
-- When will you meet?
-- Who is spearheading or point of contact?
-- How will you handle conflict?
-- Ceremonies? (do you want backlog refinement? retrospectives?)
-- Known objectives

**Reply by mrmetaverse** [reply]
for reference: the gLTF working group is a great exemplar of a working group functioning as a working group 
https://github.com/omigroup/gltf-extensions

Also the website working group which meets occasionally https://github.com/omigroup/website

**Comment by mrmetaverse** [comment]
Is open source right for the metaverse? https://trello.com/c/qIKTuq6e/5-is-open-source-right-for-the-metaverse

**Reply by mrmetaverse** [reply]
converting to discussion! (thanks RedEagle)

**Comment by mrmetaverse** [comment]
https://opencollective.com/omigroup

**Comment by mrmetaverse** [comment]
![image](https://user-images.githubusercontent.com/63426722/146240604-3bdad81a-dc1a-4117-a4d1-d59480174290.png)

**Comment by mrmetaverse** [comment]
Attendance (at end) 
![image](https://user-images.githubusercontent.com/63426722/146240719-75b8f897-3362-4702-ac42-83fdd249e599.png)
